packages/hardhat-chai-matchers/src/internal/reverted/revertedWithoutReason.ts

```diff
@@ -1,38 +1,46 @@
+import { buildAssert } from "../../utils";
 import { decodeReturnData, getReturnDataFromError } from "./utils";
 
 export function supportRevertedWithoutReason(Assertion: Chai.AssertionStatic) {
   Assertion.addMethod("revertedWithoutReason", function (this: any) {
+    // see buildAssert's jsdoc
```

Comment:
I'd like to see a one-line explanation here, something like `capture negated flag before async code executes. see buildAssert's jsdoc.`